docs_app/src/app/custom-elements/toc/toc.component.ts

```diff
@@ -1,6 +1,5 @@
 import { AfterViewInit, Component, ElementRef, OnDestroy, OnInit, QueryList, ViewChildren } from '@angular/core';
-import { asapScheduler as asap, combineLatest, Subject } from 'rxjs';
-import { startWith, subscribeOn, takeUntil } from 'rxjs/operators';
+import { asapScheduler as asap, combineLatest, Subject, startWith, subscribeOn, takeUntil } from 'rxjs';
```

Comment:
Nit: this could be fixed as well: `asapScheduler as asap` -> `asapScheduler`. Not sure if possible though.

Reply:
Next time.